sdk/documentintelligence/ai-document-intelligence-rest/review/ai-document-intelligence.api.md

```diff
@@ -12,8 +12,6 @@ import type { CreateHttpPollerOptions } from '@azure/core-lro';
 import type { HttpResponse } from '@azure-rest/core-client';
```

Comment:
I don't think we should be exposing `CreateHttpPollerOptions` to our users, it is meant to be consumed by code gen only.

Reply:
this is generated code, I did not touch, should I log an issue at autorest?

Reply:
done